packages/plugins/rubygem-openscap_parser/rubygem-openscap_parser.spec

```diff
@@ -0,0 +1,90 @@
+# template: nonscl
+%global gem_name openscap_parser
+
+Name: rubygem-%{gem_name}
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: Parse OpenSCAP content
+Group: Development/Languages
+License: MIT
+URL: https://github.com/dLobatog/openscap_parser
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: ruby(release)
+Requires: ruby
+Requires: ruby(rubygems)
+Requires: rubygem(nokogiri) >= 1.6
+Requires: rubygem(nokogiri) < 2
+BuildRequires: ruby(release)
+BuildRequires: ruby
+BuildRequires: rubygems-devel
+BuildArch: noarch
+Provides: rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+This gem is a Ruby interface into SCAP content. It can parse SCAP datastream
+files (i.e. ssg-rhel7-ds.xml), scan result files output by oscap eval, and
+tailoring files.
+
+
+%package doc
+Summary: Documentation for %{name}
+Group: Documentation
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+gem unpack %{SOURCE0}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build %{gem_name}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{_bindir}
+cp -a .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/console
+%{_bindir}/setup
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/CODE_OF_CONDUCT.md
```

Comment:
Probably good for `%files doc` and `%doc` marker. We should do that in our gem2rpm template.